src/wallet/rpcwallet.cpp

```diff
@@ -5520,14 +5520,16 @@ UniValue z_mergetoaddress(const UniValue& params, bool fHelp)
                 isToTaddr = true;
             },
             [&](libzcash::SaplingPaymentAddress addr) {
-                isToSaplingZaddr = true;
                 // If Sapling is not active, do not allow sending to a sapling addresses.
                 if (!saplingActive) {
                     throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter, Sapling has not activated");
                 }
+                estimatedTxSize += OUTPUTDESCRIPTION_SIZE;
             },
             [](libzcash::SproutPaymentAddress) { },
-            [](libzcash::UnifiedAddress) { },
+            [&](libzcash::UnifiedAddress) {
+                estimatedTxSize += ZC_ZIP225_ORCHARD_BASE_SIZE + ZC_ZIP225_ORCHARD_MARGINAL_SIZE * 2;
```

Comment:
Hmm, technically this over-estimates the transaction size if we actually end up selecting Sapling. But that's fine for now; this at least ensures it is possible to create a transaction if an Orchard-containing UA is specified.

Reply:
Ah, right – well, I can fix this if you’re already asleep. Basically, I’ll wait to see if this lands tonight (or if you say “yeah, fix it”). If not, I’ll update this (and the release notes) and push them & build before I go to sleep so this can get merged in the AM.

Reply:
Oh, it's coming back to me – it's just difficult^Wannoying to figure out which receiver we’ll select here. There are a couple follow-on changes to fix this. One is to unify with some duplicated tx estimation logic elsewhere. And once there's a proper ZTXOSelector that supports mergetoaddress, the limiting can be handled in WalletTxBuilder … at which point it's not really estimation any more.

Reply:
See my comment above. A mild overestimate is not a problem.